spec/index.html

```diff
@@ -748,6 +760,26 @@ <h2>Literals</h2>
       same literal <a>RDF terms</a> and are not
       <a>term-equal</a> because their
       <a>lexical form</a> differs.</p>
+
+    <section id="section-text-direction" class="informative">
+      <h3>Text Direction</h3>
+
+      <p>The <a>text direction</a> of a <a>language-tagged string</a>
+        provides means of establishing the direction of text
+        written or mixed with right-to-left scripts.
+        The [[[?UAX9]]] [[?UAX9]] provides support for automatically rendering
+        a sequence of characters in logical order,
+        so that they are visually ordered as expected.
+        However, this is not sufficient to correctly render bidirectional text.</p>
+
+      <p>For example, the <a>language-tagged string</a> `"פעילות הבינאום, W3C"`
+        expressed with language tag `he` will be displayed incorrectly
+        unless the <a>text direction</a> of `rtl` (for right-to-left)
+        is also provided.
+        When this is provided to a user agent,
```

Comment:
```suggestion
      <p>For example, the text `"פעילות הבינאום, W3C"` with language tag "he" may be displayed
      incorrectly unless the <a>text direction</a> of `rtl` (for right-to-left)
        is also provided.
```
or
```suggestion
      <p>For example, the text `"פעילות הבינאום, W3C"` with language tag "he" 
      will often be displayed incorrectly unless the <a>text direction</a>
      of `rtl` (for right-to-left) is also provided.
```

Reply:
I think you meant to replace the previous four lines, not the last. I'll create a substitution.